src/test/java/seedu/sourcecontrol/model/ModelManagerTest.java

```diff
@@ -146,8 +146,8 @@ public void getAssessment() {
 
     @Test
     public void equals() {
-        AddressBook addressBook = new AddressBookBuilder().withPerson(ALICE).withPerson(BENSON).build();
-        AddressBook differentAddressBook = new AddressBook();
+        SourceControl addressBook = new SourceControlBuilder().withStudent(ALICE).withStudent(BENSON).build();
```

Comment:
```suggestion
        SourceControl sourceControl = new SourceControlBuilder().withStudent(ALICE).withStudent(BENSON).build();
```
should we change these as well? same for the line below